src/react-auth0-spa.js

```diff
@@ -0,0 +1,87 @@
+import createAuth0Client from "@auth0/auth0-spa-js";
```

Comment:
Can we put this under `src/page-verification-needed`? The idea is the keep helper components within the folders where they're needed, and then "graduate" them outwards if/when other things need them as well.

If we were to bring this outwards, maybe put in `src/auth`?

Reply:
I put all of the Auth0 files in `src/auth` with a brief README.